file-read-write/file-read-write.cr

```diff
@@ -0,0 +1,28 @@
+#This is not an in-depth guide to file operations but
+#merely a basic idea how to work with text files in crystal
+
+#use this path for our file
+path = "expample.txt"
+#create a new file and open it in write mode
+#other files with the same name will be overwritten
+example_file = File.new path, "w"

```

Comment:
You should be using the [block-version](https://crystal-lang.org/api/0.23.1/File.html#open%28filename%2Cmode%3D%26quot%3Br%26quot%3B%2Cperm%3DDEFAULT_CREATE_MODE%2Cencoding%3Dnil%2Cinvalid%3Dnil%2C%26block%29-class-method) when it's available since that's a good pratice.

Reply:
I'll keep that in mind. Do you think that it should be changed in the example?

Reply:
Using block version opening a file should look like this:

```crystal
our_file = File.open path, "w" do |file|
  file.puts "Another line"
end # Automatically closes the file when the block returns
```